spring-boot-modules/spring-boot-keycloak/src/main/java/com/baeldung/keycloak/SecurityConfig.java

```diff
@@ -19,6 +25,9 @@
 @EnableWebSecurity
 class SecurityConfig {
 
+    private static final String REALM_ACCESS_CLAIM = "groups";
```

Comment:
for me, this is "realm_access", may I know how you got that value?

Reply:
Hey, for keycloack distribution in the article this is where the claims are. You can inspect the token in debug and check.

Reply:
Thanks for the clarification!